build/xtask/src/config.rs

```diff
@@ -348,6 +348,58 @@ impl Config {
             );
         }
 
+        if let Some(ext_int_ctrl_toml) = self.tasks.get("ext_int_ctrl") {
```

Comment:
Can you please add a comment here with the assumptions/expectations? This requires the `app.toml` file to have a task that is named `ext_int_ctrl` for this to work.

Reply:
I can. These expectations are in line with what an `ext_int_ctrl` (i.e. the PLIC) would need anyways. Any failure in this part would almost certainly lead to a failure when trying to build the `ext_int_ctrl` task anyways.

Reply:
Right. Just wanted to make sure that readers of xtask understand what this part of code is/does. It would be helpful also when upstreaming code since ext_int_ctrl is currently just a RV-arch related concept in hubris.

Reply:
Agreed, I'll put something down.